addon/components/basic-dropdown/content.js

```diff
@@ -117,5 +142,17 @@ export default WormholeComponent.extend({
       dropdown.removeEventListener('DOMNodeInserted', reposition);
       dropdown.removeEventListener('DOMNodeRemoved', reposition);
     }
+  },
+
+  touchMoveHandler() {
+    const appRoot = this.get('appRoot');
+    if (!this.isDestroying && !this.isDestroyed) {
+      this.hasMoved = true;
+      appRoot.removeEventListener('touchmove', this.touchMoveHandler.bind(this));
+    }
+  },
+
+  touchStartHandler () {
+    this.get('appRoot').addEventListener('touchmove', this.touchMoveHandler.bind(this));
```

Comment:
I think that this event listener should be attached on the capture phase (pass true as the last argument). Same for removing it.